README.md

```diff
@@ -23,4 +23,35 @@ We use [apidoc.js](http://apidocjs.com/) for API Documenation.
 
 To generate documentation, run the command `apidoc -i src/ -o docs/`. This command will 
 generate all documentation for files underneath the `src/` directory and generate the documentation
- in the `docs/` folder.  The `npm build` command will also generate the documentation. 
\ No newline at end of file
+ in the `docs/` folder.  The `npm build` command will also generate the documentation.
+
+## Docker Deployment
+
+A [docker](https://www.docker.com) image is built to run the application in
+production. The following commands will help you deploy Endor in a docker
+container on your local machine (for development, normally you don't need
+to do this; just run `npm start`):
+
+```bash
+# Building the image (-t gives a tag name)
+docker build -t hammerio/endor .
+# List docker images
+docker images
+# Runs the image, redirecting port 8888 on your machine to
+# the exposed port in the image 
+docker run -p 8888:3000 hammerio/endor
```

Comment:
Your Dockerfile exposes 3001 as the port, not 3000. Also, change the image tag to `hammerio/koma`